src/bionics.cpp

```diff
@@ -1335,6 +1335,11 @@ bool player::uninstall_bionic(std::string const &b_id, int skill_level)
         return false;
     }
 
+	if ( b_id == "bio_eye_optic") {
```

Comment:
> I just try to follow what everything around it looks like. What should be different?

Yeah, I see that but currently Kevin apply a lot of efforts for making entire project consistent to style described [here] (https://github.com/CleverRaven/Cataclysm-DDA/blob/master/doc/CODE_STYLE.txt).